userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Now the modern probe should also work with the sinsp-example, but let's do further tests before adding it to the CI :)